src/components/App.tsx

```diff
@@ -266,7 +266,7 @@ class App extends React.Component<any, AppState> {
   }
 
   private syncActionResult = withBatchedUpdates((res: ActionResult) => {
-    if (this.unmounted) {
+    if (this.unmounted || res === false) {
```

Comment:
`!res`?

Reply:
well that's what it was called :), short for `result`

Reply:
I think the suggestion was to use `!res` instead of `res === false` :sweat:

Reply:
Ha, you're right. Anyway, I went with `=== false` to be explicit.